go.mod

```diff
@@ -12,12 +12,13 @@ require (
 	github.com/hairyhenderson/toml v0.4.2-0.20210923231440-40456b8e66cf
 	github.com/itchyny/gojq v0.12.13
 	github.com/pkg/errors v0.9.1
+	github.com/robertkrimen/otto v0.2.1
 	github.com/stretchr/testify v1.8.4
 	github.com/ugorji/go/codec v1.2.11
 	golang.org/x/text v0.11.0
 	golang.org/x/tools v0.7.0
 	gotest.tools/v3 v3.4.0
-	k8s.io/apimachinery v0.27.3
+	k8s.io/apimachinery v0.26.4
```

Comment:
Don't bump these

Reply:
@moshloop All other repos use `v0.26.4`. Else I'd have to bump it up on canary checker

Reply:
@adityathebe  ok - then downgrade k8s.io/api as well